core/bin/snapshots_creator/src/main.rs

```diff
@@ -71,15 +87,16 @@ async fn main() -> anyhow::Result<()> {
     let _guard = builder.build();
     tracing::info!("Starting snapshots creator");
 
-    let object_store_config =
-        SnapshotsObjectStoreConfig::from_env().context("SnapshotsObjectStoreConfig::from_env()")?;
-    let blob_store = ObjectStoreFactory::new(object_store_config.0)
+    let object_store_config = general_config
```

Comment:
the correct object storage placed in `general_config.snapshot_creator.object_storage`

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2471/commits/4d581ea239c9a468d79a79c050fbfd3ce907d91e